### PR TITLE
Added experimental red hat 7 stig module with 10 checks implemented

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: cfbs.json [hunk 1]
```diff
@@ -1153,6 +1153,99 @@
...
+      "version": "0.0.1",
+      "commit": "adf4fd65eeddf2007c4cd0592955d39323dc191e",
+      "subdirectory": "stig/red_hat_enterprise_linux_7/",
+      "dependencies": ["compliance-report-imports"],
+      "steps": [
+        "copy ./compliance-reports/compliance-report.json ./.no-distrib/compliance-report-definitions/red-hat-enterprise-linux-7-security-technical-implementation-guide.json",
+        "copy ./config.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/config.cf",
+        "copy ./V-204392/V-204392.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204392.cf",
+        "copy ./V-204424/V-204424.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204424.cf",
+        "copy ./V-204425/V-204425.cf services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204425.cf",
```
Comment (olehermanse) [inline cfbs.json:1170]: Why all the individual `copy`'s?

File: cfbs.json [hunk 2]
```diff
@@ -1153,6 +1153,99 @@
         "json cfbs/def.json def.json"
       ]
     },
+    "stig-red-hat-enterprise-linux-7": {
```
Comment (olehermanse) [inline cfbs.json:1156]:
```suggestion
    "stig-rhel-7": {
```

File: cfbs.json [hunk 3]
```diff
@@ -1153,6 +1153,99 @@
...
+      "description": "Red Hat Enterprise Linux 7 Security Technical Implementation Guide",
+      "tags": ["security", "experimental"],
+      "repo": "https://github.com/nickanderson/cfengine-security-hardening",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "adf4fd65eeddf2007c4cd0592955d39323dc191e",
+      "subdirectory": "stig/red_hat_enterprise_linux_7/",
```
Comment (olehermanse) [inline cfbs.json]:
```suggestion
      "subdirectory": "stig/red_hat_enterprise_linux_7",
```

File: cfbs.json [hunk 4]
```diff
@@ -1153,6 +1153,99 @@
...
+        "policy_files services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-214799.cf",
+        "policy_files services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204497.cf",
+        "policy_files services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204443.cf",
+        "policy_files services/cfbs/modules/stig-red-hat-enterprise-linux-7/V-204442.cf",
+        "policy_files services/cfbs/modules/stig-red-hat-enterprise-linux-7/chaos.cf",
+        "policy_files services/cfbs/modules/stig-red-hat-enterprise-linux-7/config.cf",
+        "bundles RHEL_7_STIG:run",
+        "input ./input.json def.json"
+      ],
+      "input": [
```
Comment (olehermanse) [inline cfbs.json:1198]: Input looks good 🚀 😄

File: cfbs.json (hunk 1 above)
Comment (nickanderson) [reply cfbs.json:1170]: Why not?

Comment (olehermanse) [reply cfbs.json:1170]: Looks tedious and error-prone to maintain

File: cfbs.json (hunk 4 above)
Comment (nickanderson) [reply cfbs.json:1198]: Whoa, amazing.

File: cfbs.json (hunk 2 above)
Comment (nickanderson) [reply cfbs.json]: ack.

File: cfbs.json [hunk 5]
```diff
@@ -1153,6 +1153,99 @@
         "json cfbs/def.json def.json"
       ]
     },
+    "stig-rhel-7": {
+      "description": "Red Hat Enterprise Linux 7 Security Technical Implementation Guide",
+      "tags": ["security", "experimental"],
```
Comment (olehermanse) [inline cfbs.json:1158]:
```suggestion
      "tags": ["security", "compliance", "experimental"],
```

**Review by olehermanse — APPROVED** [review]

**Pull request merged**